src/glados/audio_io/sounddevice_io.py

```
@@ -0,0 +1,236 @@
+        the VAD model and placed in the sample queue.
+
+        Raises:
+            RuntimeError: If the audio input stream cannot be started
+            sd.PortAudioError: If there's an issue with the audio hardware
+        """
+        if self.input_stream is not None:
+            self.stop_listening()
+
+        def audio_callback(
+            indata: np.dtype[np.float32],
+            frames: int,
```

Pylint (3.3.7)

[warning] Unused argument 'frames'

(W0613)